tests/platforms/iOS/xcode/test_run.py

```diff
@@ -72,7 +72,7 @@ def test_run_multiple_devices_input_disabled(run_command, first_app_config):
         run_command.run_app(first_app_config, test_mode=False, passthrough=[])
 
 
-def test_run_app_simulator_booted(run_command, first_app_config, tmp_path):
+def test_run_app_simulator_booted(run_command, underscore_app_config, tmp_path):
```

Comment:
My assumption is that this has been changed to `underscore_app_config` to piggyback a test of underscore handling onto an existing test. That makes some sense - however, it's not immediately obvious that the test is serving a double purpose here. If a future iteration of the code didn't ned to test booting the simulator (or needed to test it a different way), it would be easy to lose this testing case. 

Even though there will be some duplicated code, I'd suggest making an explicit "run_app_with_underscore" test.